tests/foreman/ui/test_repository.py

```diff
@@ -920,10 +921,18 @@ def test_negative_download_policy_displayed_for_non_yum_repo(self):
 
         @Assert: Dropdown for download policy is not displayed for non-yum repo
         """
-        non_yum_repo_types = [
-            repo_type for repo_type in REPO_TYPE.values()
-            if repo_type != 'yum'
-        ]
+        os_version = get_host_os_version()
```

Comment:
The change is only applicable for a single test as we might want to run same test for rhel7 with all other options but on rhel6 w/o ostree so I perfer the change as is. we may move this to `setUpclass` later if other tests needs to be updated, which is not the case for now.